letsencrypt/rootfs/etc/cont-init.d/file-structure.sh

```diff
@@ -34,6 +34,16 @@ echo -e "dns_cloudxns_api_key = $(bashio::config 'dns.cloudxns_api_key')\n" \
       "aws_access_key_id = $(bashio::config 'dns.aws_access_key_id')\n" \
       "aws_secret_access_key = $(bashio::config 'dns.aws_secret_access_key')\n" \
       "dns_sakuracloud_api_token = $(bashio::config 'dns.sakuracloud_api_token')\n" \
-      "dns_sakuracloud_api_secret = $(bashio::config 'dns.sakuracloud_api_secret')" > /data/dnsapikey
+      "dns_sakuracloud_api_secret = $(bashio::config 'dns.sakuracloud_api_secret')\n" \
+      "certbot_dns_transip:dns_transip_username = $(bashio::config 'dns.transip_username')\n" \
+      "certbot_dns_transip:dns_transip_key_file = /data/transip-rsa.key\n" > /data/dnsapikey
 
 chmod 600 /data/dnsapikey
+
+## Prepare TransIP RSA key
+if bashio::config.exists 'dns.transip_api_key'; then
+      echo "$(bashio::config 'dns.transip_api_key')" > /data/transip-api.key
+      chmod 600 /data/transip-api.key
+      openssl rsa -in /data/transip-api.key -out /data/transip-rsa.key
```

Comment:
Since `/data/transip-api.key` does not seem to be used, could this be
```
echo "$(bashio::config 'dns.transip_api_key')" | openssl rsa -out /data/transip-rsa.key
```

Reply:
So true, thanks.